### DISCUSSION TITLE
Why Wix4/Wix5 Package element doesn't contain Platform attribute?

### DISCUSSION THREAD

**Discussion by talbotsk** [wixtoolset/issues#8583] [Questions] [answered]
See difference between:
https://wixtoolset.org/docs/v3/xsd/wix/package/
and
https://wixtoolset.org/docs/schema/wxs/package/
How can I specify Platform for MSI package in Wix4?

**Comment by wmanning** [comment, marked as answer]
In the "WiX v4 for WiX v3 users" FAQ: 

> with with the -arch switch at the wix.exe command line or the equivalent MSBuild property InstallerPlatform.

**Comment by talbotsk** [comment]
Now I know that Platform attribute was removed in Wix4 and can not be used anymore. Instead of that platform must be specified as wix.exe -arch x64 parameter. Thank you for your answer.
